styles.css

```diff
@@ -1,50 +1,126 @@
 
 
-
-
 body {
   background-color: #c1cdcd;
   margin: 0;
   padding: 0;
-  font-family: Georgia, 'Times New Roman', Times, serif;
   display: grid;
-  grid-template-columns: 1fr 3fr;
+  grid-template-columns: 1fr 4fr;
   grid-template-areas: "sidebar content";
+  font-family: Caveat, Georgia, 'Times New Roman', Times, serif;
+  font-size: 1.1em;
 }
 
 nav {
-  background-color: #c1ffc1;
+  background-color: #ffefdb;
   grid-area: sidebar;
   display: grid;
   grid-template-columns: 1fr;
   grid-template-rows: 40px auto;
   min-height: 100vh;
+  min-width: 270px;
+  font-family: "Grenze Gotisch", Times, serif;
 }
 
 nav>header {
-  background-color: #7ccd7c;
+  background-color: #8b2323;
+  padding: 10px 10px 20px 10px;
+  min-height: 1em;
+  font-size: 1.6em;
+  font-weight: 600;
+  color: #ffec8b;
+}
+
+nav>ul {
+  margin-top: 2.5em;
 }
 
 nav>footer {
-  background-color: #adcd7c;
-  height: 60px;
-  font-size: 80%;
+  background-color: #ab4343;
+  height: 4em;
+  padding: 0px 10px 10px 10px;
+  font-size: 0.8em;
+  color: #fff;
   align-self: end;
 }
 
 main {
-  background-color: #fdf5e6;
+  background-color: #8b7d6b;
   grid-area: content;
+  padding: 0 15px;
 }
 
 section {
-  background-color: #eec591;
+  background-color: #ffefdb;
+  margin: 5px 0 20px 0;
+  padding: 1em;
+  border-radius: 5px;
+  box-shadow: 0 5px 10px rgba(0, 0, 0, 0.6);
+  color: #4b3305;
 }
 
 section>header {
   background-color: #8b3e2f;
+  padding: 5px 10px;
+  margin: -0.6em -0.6em 0.6em -0.6em;
+  border-radius: 2px;
+  font-family: "Grenze Gotisch", Times, serif;
+  font-size: 1.4em;
+  font-weight: 600;
+  color: #fdd891;
+}
+
+article p:first-child::first-letter {
```

Comment:
As you can see, when the screen is really wide and the text remains in a single line, the capitalised first letter doesn't look as good as when you have 3 lines in the paragraph. 
![image](https://user-images.githubusercontent.com/19556565/90322782-5a162580-df50-11ea-92e8-3dcef9442d74.png). Could you try to make it look better for this case?

Reply:
Oh, damn, it's true. Plus it will look bad whenever the text is too short for the section.. I'm searching for methods to count the lines in a paragraph element but they all rely on Javascript. For now I think I'll use a media query keeping the use of drop caps for not-so-wide viewports, at least until I get more familiar with JS.